internal/datanode/flow_graph_insert_buffer_node_test.go

```diff
@@ -560,3 +562,137 @@ func TestFlowGraphInsertBufferNode_AutoFlush(t *testing.T) {
 
 	})
 }
+
+// CompactedRootCoord has meta info compacted at ts
+type CompactedRootCoord struct {
```

Comment:
dup to RootCoordFactory  in  mock_test.go

Reply:
CompactedRootCoord composes with `RootCoordFactory` when used and only override `DescribeCollection` behavior